src/main/java/teammates/logic/api/Logic.java

```diff
@@ -144,6 +144,15 @@ public void putInstructorDocuments(List<InstructorAttributes> instructors) {
         instructorsLogic.putDocuments(instructors);
     }
 
+    /**
+     * Update instructor being edited to ensure validity of the course
+     * 
+     * @see InstructorsLogic#updateToEnsureValidityOfInstructorsForTheCourse(String, InstructorAttributes) 
+     */
+    public void updateToEnsureValidityOfInstructorsForTheCourse(String courseId, InstructorAttributes instructorToEdit) {
```

Comment:
Although documentation is important, it is not what I was referring to in the previous comment. The "missing thing" is still missing.